stdlib/InteractiveUtils/src/macros.jl

```diff
@@ -83,6 +83,15 @@ macro which(ex0::Symbol)
     return :(which($__module__, $ex0))
 end
 
+macro code_typed(optimize, ex0)
```

Comment:
Switch argument order here? Keyword arguments are usually last and `@code_typed foo(x) optimize=false` looks better than `@code_typed optimize=false foo(x)` IMO.

Reply:
I prefer the argument order as is, CUDAnative has the same ordering

Reply:
I went with the kwargs first because that makes it easier to place a `@code_something foo=bar` before an expression without having to edit at both ends.